app/lib/MyAccountsStore.js

```diff
@@ -83,6 +83,25 @@ export default class MyAccountsStore {
         : this.addAccount(konnector, account.values)
     result
     .then(account => {
+      return cozy.client.fetchJSON('PATCH', konnector.links.permissions, {
+        data: {
+          id: konnector.links.permissions,
+          type: 'io.cozy.permissions',
+          attributes: {
+            type: 'app',
+            source_id: konnector._id,
+            permissions: {
+              saveFolder: {
+                type: 'io.cozy.files',
+                values: [folder._id]
+              }
+            }
+          }
+        }
+      })
+      .then(() => account)
```

Comment:
Looks weird, because if I am understanding correctly, the `account` returned here is the same next passed as parameter line 104. Am I missing something ?